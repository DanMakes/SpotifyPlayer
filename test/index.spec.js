import React from 'react';
import SpotifyWebPlayer from '../src'; // eslint-disable-line import/no-unresolved

describe('SpotifyWebPlayer', () => {
  let wrapper;

  beforeAll(() => {
    wrapper = mount(<SpotifyWebPlayer />);
  });

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
