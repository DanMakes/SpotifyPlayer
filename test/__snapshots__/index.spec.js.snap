// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotifyWebPlayer should render properly 1`] = `
<SpotifyWebPlayer
  name="Spotify Web Player"
>
  <div
    className="rswp"
  >
    <div
      className="rswp__container"
    >
      <div
        className="rswp__loader"
      >
        <div />
      </div>
    </div>
  </div>
</SpotifyWebPlayer>
`;
